components/org.wso2.carbon.identity.sso.saml/src/main/java/org/wso2/carbon/identity/sso/saml/dto/SAMLSSOSessionDTO.java

```diff
@@ -55,6 +55,7 @@
     private String requestedAuthnContextComparison;
     private List<ClaimMapping> requestedAttributes;
     private Properties properties;
+    private String loginTenantDomain;
```

Comment:
loggedInTenantDomain makes much sense. Doesn't it?

Reply:
Addressed with https://github.com/wso2-extensions/identity-inbound-auth-saml/pull/348